source/_integrations/dsmr_reader.markdown

```diff
@@ -33,6 +33,16 @@ To use this DSMR Reader sensor integration, you need to have a DSMR Reader insta
 
 {% include integrations/config_flow.md %}
 
+## Configuring the Energy Dashboard
+
+It is most advisable to use the `sensor.dsmr_reading_*` sensors in the energy dashboard over the "merged" and "day" sources. The "reading" sensors provide the most stable source of data for Home Assistant to use. These MQTT values are part of the "Telegram: Split topic" MQTT values within DSMR Reader, so make sure to enable it.
```

Comment:
Why does that matter? (reading, merged, day) Every source that meets the requirements is viable for use, the result should make any difference?

Reply:
The reading values contain a running total, while the day values get reset each day at midnight. In some cases, there might be some data loss at that time. This is especially true when the Home Assistant instance was down for a day or more.

Slightly tangential: after your change to the state classes, the other ones actually aren't viable sources any more. I still need to revert that change, as they recently fixed the bug in DSMR Reader.